src/lib.rs

```diff
@@ -299,7 +297,7 @@ pub fn build_tcp_ws_secio_mplex_yamux(keypair: identity::Keypair)
         .authenticate(secio::SecioConfig::new(keypair))
         .multiplex(core::upgrade::SelectUpgrade::new(yamux::Config::default(), mplex::MplexConfig::new()))
         .map(|(peer, muxer), _| (peer, core::muxing::StreamMuxerBox::new(muxer)))
-        .timeout(Duration::from_secs(20)))
+        .timeout(std::time::Duration::from_secs(20)))
```

Comment:
Nit pick: Seems like we always import `Duration` at the top. Why change that?

Reply:
To fix an unused warning when compiled with no-default-features

Reply:
Ah, yes, you mentioned that on the top. Sorry for the noise.